docs/build/isc/v1.3/docs/tutorials/defi-lend-borrow-tutorial-part-1.md

```diff
@@ -0,0 +1,289 @@
+- **Parameters:**
+  - `account`: The account to check.
+  - `iToken`: The token in question.
+  - `redeemTokens`: The number of tokens to redeem.
+  - `borrowTokens`: The number of tokens to borrow.
+- **Returns:** 
+  - `totalAccountCollaterals`: The total collateral balance.
+  - `totalAccountBorrows`: The total borrow balance.
+
+```solidity reference
+https://github.com/iota-community/Defi-lend-borrow/blob/3a368391f4767d1decb209ad6bfdd20a2b08fa03/contracts/ITokenManager.sol#L187-L247
+
+```
+- `updateTokenUSDPrice(address token, uint256 newUSDPrice)`: Updates the USD price of a supported token.
+
+- **Parameters:**
+  - `token`: The address of the token.
+  - `newUSDPrice`: The new USD price for the token.
+- **Modifiers:** `onlyOwner`, `onlySupportedToken`
+- **Reverts:** `TokenNotListed` if the token is not supported.
+
+```solidity reference
+https://github.com/iota-community/Defi-lend-borrow/blob/bfb658e82611657a50885a2509f21a07da2bbef1/contracts/ITokenManager.sol#L245-L258
+
+```
+
+- `getTokenUSDPrice(address token) → uint256`: Returns the USD price of a supported token.
+
+- **Parameters:**
+  - `token`: The address of the token.
+- **Returns:** The USD price of the token as a `uint256`.
+
+```solidity reference
+https://github.com/iota-community/Defi-lend-borrow/blob/3a368391f4767d1decb209ad6bfdd20a2b08fa03/contracts/ITokenManager.sol#L249-L262
+
+```
+
+We have now covered all relevant parts and the functionality of the `ITokenManager` contract. For reference, you can follow the [full contract code](https://github.com/iota-community/Defi-lend-borrow/blob/main/contracts/ITokenManager.sol).
+
+### Underlying Token
+
+The `MockERC20` contract is a simple ERC20 token used for testing purposes. It allows minting of an initial supply to the deployer. You can add this token to the `ITokenManager` and mint new ITokens by depositing the MockERC20 tokens as the underlying Token.
+
+
+## Scripts
+
+### Compile Your Contracts
+
+We can compile our contracts created above using the below command. This command compiles your Solidity smart contracts, checking for errors and generating necessary artifacts, including the ABI (Application Binary Interface) and bytecode. These artifacts are stored in the artifacts/ directory and are essential for deploying and interacting with the contracts.
+
+```bash
+npx hardhat compile
+
+```
+
+First, create a `scripts` folder in the root of the project and add the following files under it:
+
+### Deploy script
+
+The `deploy.js` script will deploy the contract to the ShimmerEVM Testnet.
+```solidity reference
+https://github.com/iota-community/Defi-lend-borrow/blob/e227092ef1123cf8cff766ee4c786a0c40a62bf3/scripts/deploy.js#L1-L51
+
+```
+
+This will deploy the Defi Lend borrow contract to the ShimmerEVM Testnet. run it by executing:
+```bash
+npx hardhat run scripts/deploy.js --network shimmer_evm_testnet
+
+```
```

Comment:
````suggestion

```bash
npx hardhat run scripts/deploy.js --network shimmer_evm_testnet
```
````

Reply:
same for all code-blocks. Please leave a blank line between to separate MD components such as headers, codeblocks, images, etc. ; and there's no need for a blank line at the end of the codeblock